extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -118,6 +122,7 @@ public static function onPortableInfoboxNodeImageGetData( $title, &$ref, $alt )
 		wfProfileIn( __METHOD__ );
 		if ( $title ) {
 			$details = WikiaFileHelper::getMediaDetail( $title, self::$mediaDetailConfig );
+			$details['context'] = 'infobox-big';
```

Comment:
I wonder if it'd make sense to make possible contexts constants in ArticleAsJson extension. Obviously, this is the first explicit context we are introducing, so it's up to your discretion how to handle it. My two sentiments would be to either extract const or leave a todo to address when another context is added.